projects/plugins/crm/includes/ZeroBSCRM.DAL3.ObjectLayer.php

```diff
@@ -44,6 +44,18 @@ class zbsDAL_ObjectLayer {
     // e.g. Invoice object type may be commonly linked to 'contact' or 'company' object types
     protected $linkedToObjectTypes = array();
 
+	/** This field is used to store name clashes so we can change custom field names
+	 * with name clashes in function `fix_name_clash_if_needed()`
+	 * See: https://github.com/Automattic/zero-bs-crm/issues/3477
+	 *
+	 * @var array
+	 */
+	protected $name_clashes_temp_fix = array();
+
+	/** Suffix used to fix name clashes
+	 * See: https://github.com/Automattic/zero-bs-crm/issues/3477
+	 */
+	protected const NAME_CLASH_FIX_SUFFIX = '_zbs-avoid-name-clash-temporary-fix';
```

Comment:
The suffix seems large. Could we use something shorter? Maybe a numeric suffix could work. I'm thinking of the usage of the field name, like in templates.

Reply:
Made it a little shorter in e5ce479